test/types_test.rb

```diff
@@ -1,5 +1,37 @@
 require 'test_helper'
 
+describe Slop::StringOption do
+  before do
+    @options = Slop::Options.new
+    @age     = @options.string "--name"
+    @minus   = @options.string "--zipcode"
+    @result  = @options.parse %w(--name Foo --zipcode 12345)
+  end
+
+  it "returns the value as a string" do
+    assert_equal "Foo", @result[:name]
+    assert_instance_of String, @result[:name]
+    assert_equal "12345", @result[:zipcode]
+    assert_instance_of String, @result[:zipcode]
```

Comment:
These `assert_instance_of` calls are superfluous, since the type is already being implicitly asserted in your previous tests. I think having the 2 `assert_equal` calls is fine (same thing in your `SymbolOption` tests